specification/advisor/resource-manager/Microsoft.Advisor/preview/2025-08-01-preview/advisor.json

```diff
@@ -0,0 +1,1992 @@
+          "type": "string"
+        },
+        "metadata": {
+          "description": "The recommendation metadata.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          }
+        },
+        "recommendationTypeId": {
+          "description": "The recommendation-type GUID.",
+          "type": "string"
+        },
+        "risk": {
+          "description": "The potential risk of not implementing the recommendation.",
+          "enum": [
+            "Error",
+            "Warning",
+            "None"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "risk",
+            "modelAsString": true
+          }
+        },
+        "shortDescription": {
+          "$ref": "#/definitions/ShortDescription",
+          "description": "A summary of the recommendation."
+        },
+        "suppressionIds": {
+          "description": "The list of snoozed and dismissed rules for the recommendation.",
+          "type": "array",
+          "items": {
+            "format": "uuid",
+            "type": "string"
+          }
+        },
+        "extendedProperties": {
+          "description": "Extended properties",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "resourceMetadata": {
+          "$ref": "#/definitions/ResourceMetadata",
+          "description": "Metadata of resource that was assessed"
+        },
+        "description": {
+          "description": "The detailed description of recommendation.",
+          "type": "string"
+        },
+        "label": {
+          "description": "The label of recommendation.",
+          "type": "string"
+        },
+        "learnMoreLink": {
+          "description": "The link to learn more about recommendation and generation logic.",
+          "type": "string"
+        },
+        "potentialBenefits": {
+          "description": "The potential benefit of implementing recommendation.",
+          "type": "string"
+        },
+        "actions": {
+          "description": "The list of recommended actions to implement recommendation.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "additionalProperties": {
+              "type": "object"
+            }
+          },
+          "x-ms-identifiers": []
+        },
+        "remediation": {
+          "description": "The automated way to apply recommendation.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          }
+        },
+        "exposedMetadataProperties": {
+          "description": "The recommendation metadata properties exposed to customer to provide additional information.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          }
+        },
+        "tracked": {
```

Comment:
>tracked [](http://example.com/codeflow?start=9&length=7)

Enums are suggested over booleans.

Can this be modelled as a enum of saym TrackingState, with values of Enabled, Disabled etc ?